### PR TITLE
Version 1.3.0

### PR DESCRIPTION
**Pull request by bbayles**
This PR:
* Adds the `urltestdata.json` tests to ensure that these bindings match the behavior of `ada`
* Updates the version number to 1.3.0 for release

### CODE REVIEW

File: tests/test_ada_url.py [hunk 1]
```diff
@@ -394,3 +398,31 @@ def test_idna_encode(self):
...
+
+        for i, item in enumerate(test_data, 1):
+            # Skip the comments
+            if isinstance(item, str):
+                continue
+
+            # Skip tests that can't be represented properly with the json module
+            try:
+                (item.get('input') or '').encode('utf-8')
+                (item.get('base') or '').encode('utf-8')
```
Comment (anonrig) [inline tests/test_ada_url.py:416]: base can be "null" as a string.

Comment (bbayles) [reply tests/test_ada_url.py:416]: Per the [readme](https://github.com/web-platform-tests/wpt/tree/master/url#urltestdatajson):

```
The format of resources/urltestdata.json is a JSON array of comments as strings and test cases as objects. The keys for each test case are:

    input: a string to be parsed as URL.

    base: null or a serialized URL (i.e., does not fail parsing).

```

Surely this means the `"null"` the string is meant to be interpreted as a string, right?

---

In any case, this test is simply to identify test case 410, the one with the questionably-encoded-in-JSON Unicode surrogates. The actual parse is done below.

Comment (anonrig) [reply tests/test_ada_url.py:416]: My mistake!

File: update-wpt.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```
Comment (anonrig) [inline update-wpt.sh:1]: I recommend putting this under `.github` folder and also adding a github workflow similar to: https://github.com/ada-url/ada/blob/main/.github/workflows/wpt-updater.yml

Comment (bbayles) [reply update-wpt.sh:1]: I can make this change in another PR. I'd like to merge this one and do a release so @y3ti can get his ARM binaries (ref #30). Good with that?

Comment (anonrig) [reply update-wpt.sh:1]: I'm good with all decisions you make in this project. I see you as the owner, and me as the "reviewer" making recommendations (not dictating them) 👍

**Pull request merged**